build.gradle

```diff
@@ -250,17 +212,33 @@ publishing {
     }
   }
   repositories {
-    // Credentials shall be stored in ~/.gradle/gradle.properties
-    mavenRepos.each { repo ->
-      maven {
-        name repo.name
-        url repo.url
+    maven {
+      name 'project'
+      url "$buildDir/maven-repo"
+    }
+
+    maven {
+      name 'eclipse'
+      url 'https://repo.eclipse.org/content/repositories/golo-' + (version.endsWith('-SNAPSHOT') ? 'snapshots/' : 'releases/')
+      if (project.hasProperty('eclipseRepoUsername')) {
         credentials {
```

Comment:
...but in that case you get publishing tasks for both Eclipse and Bintray, right? The previous implementation from @jbnizet does not have this issue.